tests/helper_test.py

```diff
@@ -129,6 +129,18 @@ def test_unicode(self):
         expected = ['echo "foo"', 'sleep 1']
         self._test(base, expected)
 
+    @unittest.expectedFailure
+    def test_doublequoted_semicolons(self):
+        base = 'python -c "import alot; print(alot)"; echo "done"'
+        expected = ['python -c "import alot; print(alot)"', ' echo "done"']
+        self._test(base, expected)
+
+    @unittest.expectedFailure
+    def test_singlequoted_semicolons(self):
+        base = "python -c 'import alot; print(alot)'; echo 'done'"
```

Comment:
These are broken on archlinux, `python` points to `python3`, and `alot` isn't available for python3. Can we either use a stdlib package like `os`, or explicitly use `python2`?